site.js

```diff
@@ -0,0 +1,8 @@
+function showShareButtons() {

```

Comment:
*this comment is a matter of taste*

```js
function showShareButtons() {
  var shareButtonContainer = document.getElementsByClassName("share-button-container")[0];
  var newStyle = shareButtonContainer.style.display == 'none' ? 'inline-block' : 'none';
  
  shareButtonContainer.style.display = newStyle;
}
```

**PRO-TIP**: add a transition to the effect of displaying and hide the element